firebase-crashlytics/src/main/java/com/google/firebase/crashlytics/internal/persistence/CrashlyticsReportPersistence.java

```diff
@@ -222,6 +282,25 @@ private static String readTextFile(File file) {
     }
   }
 
+  /**
+   * Deletes files from the list until the list size is equal to the maximum. If list is already
+   * correctly sized, no files are deleted. List should be sorted in the order in which files should
+   * be deleted.
+   *
+   * @return the number of files retained on disk
+   */
+  private static int capFilesCount(List<File> files, int maximum) {
```

Comment:
Is it worth having a unit test on this method?

Reply:
Potentially, but I'll come back to it in another PR.